RecoTracker/MkFit/plugins/MkFitHitConverter.cc

```diff
@@ -185,19 +220,37 @@ void MkFitHitConverter::convertHits(const HitCollection& hits,
       err.At(0, 2) = gerr.czx();
       err.At(1, 2) = gerr.czy();
 
+      auto clusterRef = hit.firstClusterRef();
+      if UNLIKELY (clusterRef.id() != clusterID) {
```

Comment:
does `[UN]UNLIKELY` matter anymore?
I've seen that on more recent (10 years perhaps) the hardware wouldn't care because the choice is dynamic and it's just as effective without the hints.

Reply:
Good question, I don't know.

Reply:
while looking at C++20 features I learned of `[[likely]]` and `[[unlikely]]`
http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2018/p0479r5.html
there are supported in gcc 9, apparently.

So, it looks like at least formally this feature is still alive